managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+   This page lists, and provides a summary of, the credentials for accessing and using embedded workflows. From this page, you can add, update, and delete credentials.
+
+   These credentials are tied to the provider instance, and are used by the workflow when it runs. The credential types are defined by the provider.
+
+#### UI Field Updates
+{: #field_updates}
+
+Existing integration points are updated for the service catalog creation and management process to support choosing to use **Embedded Workflows** or **Embedded Automate**. This choice can be set when working with service catalog items and service dialogs.
+
+* Service Dialog
+
+   When creating a service dialog, fields that are dynamic now include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Service Dialog options](../images/embedworkflow_servicedialogdynamic.png){: caption="Service Dialog options" caption-side="bottom"}
+
+* Service Catalog Items
+
+   The provisioning, configuring, or retiring entry points now include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Entry Points options](../images/embedworkflow_entrypointsembedded.png){: caption="Entry Points options" caption-side="bottom"}
+
+### Adding an Embedded Workflows Repository
+
+As a starting point to begin authoring a workflow, you must first add the repository that includes your workflows.
+
+To add a repository, complete the following steps:
+
+1. Click  **Automation** > **Embedded Workflows** > **Repositories** > **Configuration** > **Add New Repository**.
+
+   ![Add a repository](../images/embedworkflow_addnewrepository.png){: caption="Add a repository" caption-side="bottom"}
+
+2. Enter *Name*, *Description*, and a *GitHub URL*.
+
+3. Provide a Repository Name in the **Name** field.
+
+4. Add a description for the repository in the **Description** field.
+
+5. Add a **GitHub URL** for the repository.
+
+6. Select the appropriate SCM (Source Control) credentials from the **SCM Credentials** dropdown menu. These are the credentials for how you access GitHub, and are used in different ways within embedded workflows.
+
+7. Provide a branch name in the **SCM Branch** field for the branch that you want to pull.
+
+8. Click **Save**. Upon saving, the repository is imported and scanned for any included workflows.
+
+   ![Repository details](../images/embedworkflow_addnewrepository_details.png){: caption="Repository details" caption-side="bottom"}
+
+9. Confirm that the synchronization with your repository is successful by viewing the list of workflows that are detected for the repository.
+
+**Note:** If you are adding a repository that is not publicly available, add an SCM (source control) credential for accessing the repository. This credential is separate from the workflow credentials that you configure in the procedure for adding the repository. Source Control credentials have several attributes that can be configured:
```

Comment:
> This credential is separate from the workflow credentials that you configure in the procedure for adding the repository.

Hm?  When adding a repository you would be optionally adding an SCM credential not a Workflow credential

Reply:
I feel like this should be merged into line 84 above https://github.com/ManageIQ/manageiq-documentation/pull/1752/files#diff-a944ad335565b3b6f0c1776d24ce585d3b42531e2d45bc20a5eaab1950a871e4R84

Reply:
I have merged this content as suggested.